library/core/class.authenticationprovidermodel.php

```diff
@@ -163,6 +163,28 @@ public static function getProviderByScheme($authenticationSchemeAlias, $userID =
         return false;
     }
 
+
+    /**
+     * Get a list of providers by (read type) scheme.
+     *
+     * @param $authenticationSchemeAlias
+     * @return array
+     */
+    public function getProvidersByScheme($authenticationSchemeAlias) {
```

Comment:
Since we're already in the "Authentication Provider" model, this should probably be shortened to `getByScheme`.

Reply:
I'm following the current convention. Check the method above.